src/main/java/transforms/BooleanPrefixTransform.java

```diff
@@ -54,17 +54,17 @@ public String getColumn() {
     }
 
     @Override
-    public boolean applyCondition( Map<String, String> row ) {
+    public boolean applyCondition( Map<String, Object> row ) {
 
         if ( !( row.containsKey( column ) ) ) {
             throw new IllegalStateException( String.format( "The column %s is not found.", column ) );
         }
 
         final String o;
         if ( this.ignoreCase ) {
-            o = row.get( column ).toLowerCase();
+            o = row.get( column ).toString().toLowerCase();
```

Comment:
is row.get( column ) guaranteed to be non-null?

Reply:
(same question for the other transforms)

Reply:
Added checks in booleanTransforms.
Also added checks for nulls in main class Transformation and removed them from individual transformations here: https://github.com/openlattice/shuttle/pull/60
(would be too messy on this branch)